client/controllers/incidentReports/incidentTable.coffee

```diff
@@ -0,0 +1,35 @@
+Template.incidentTable.onCreated ->
+  @rowsSelected = new ReactiveVar(false)
+
+selectedIncidents = (event, instance) ->
+  # return all the selected incident-ids
+  $('table.incident-table tr.active td.count').map(() ->
+    $(this).data("incident-id")
+  ).get()
+
+changeIncidentStatus = (status) ->
+  incidentIds = selectedIncidents()
+  $.each incidentIds, (index, incidentId) ->
+    Meteor.call 'getIncidentReport', incidentId, (error, incident) ->
```

Comment:
I think that's because the IRs in the local collection immediately after processing have all the raw properties. Maybe we should change up the `createIncidentReportsFromEnhancements` to always return an object with the necessary properties picked? (https://github.com/ecohealthalliance/eidr-connect/blob/master/methods/incidentReports.coffee#L270-L273)
This may be the best way—increases consistency and decreases complexity (assuming we don't need the raw props somewhere else)